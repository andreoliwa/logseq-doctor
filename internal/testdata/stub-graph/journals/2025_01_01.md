- Add content under this block with some [[page]] or #tag
- Some content after the header
