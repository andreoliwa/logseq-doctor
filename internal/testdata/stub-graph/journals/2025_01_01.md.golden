- Add content under this block with some [[page]] or #tag
	- Line 1
	  Line 2
	  Line 3
- Some content after the header